bots/elasticsearch/util.py

```diff
@@ -233,4 +263,8 @@ def fetch_creativework(pks):
                 populate_types(data)
                 data['date'] = (data['date_published'] or data['date_updated'] or data['date_created'])
 
+                for field in ('affiliations', 'contributors', 'funders', 'publishers'):
```

Comment:
`affiliations` is constructed in python from the agent-work relations that don't fit in another field, plus all the contributors' affiliations. When multiple contributors are affiliated with the same organization, it has dupes, which seemed wrong. Then once I was deduping one list, I figured I may as well do all of them for good measure.

I guess duplicate affiliations aren't a real problem, though... their effect on relevance might even be desirable. More affiliations == more relevant.